src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/roslyn3.11.rd.xml

```diff
@@ -0,0 +1,7 @@
+<Directives>
+  <Application>
+    <!-- NB Shared specializations must also be replicated in the roslyn4.4.rd.xml file -->
```

Comment:
I assume we'll need to copy/paste the contents of the other XML here to get a successful test pass?

Reply:
The Roslyn3.11 project only runs a subset of tests, so specializations are not necessary. Left an RD file as a placeholder in case that changes in the future.